docs/markdown/FAQ.md

```diff
@@ -432,7 +432,7 @@ sources in the build target:
 libfoo_gen_headers = custom_target('gen-headers', ..., output: 'foo-gen.h')
 libfoo_sources = files('foo-utils.c', 'foo-lib.c')
 # Add generated headers to the list of sources for the build target
-libfoo = library('foo', sources: libfoo_sources + libfoo_gen_headers)
+libfoo = library('foo', sources: libfoo_sources + libfoo_gen_headers.to_list())
```

Comment:
Actually this should be `sources: [libfoo_sources, libfoo_gen_headers]`. People should not need to have to use `to_list` for these things, that would be bad usability.

Reply:
I agree, it does look a lot cleaner. I added a new commit to update it accordingly.
Originally I wanted to amend the original commit, but not sure how to do so in github.
Hope this is fine, otherwise I can start out fresh and make a new, corrected pull request with only one commit.

Reply:
I can squash it for now. Just remember in the future that doc-only changes should have `[skip ci]` at the end of their commit messages so CI is not run.